connections/tests/utils.py

```diff
@@ -46,3 +62,45 @@ def get_elastic_apartments_for_sale_project_uuids() -> list:
     for hit in scan:
         uuids.append(str(hit.project_uuid))
     return uuids
+
+
+def publish_elastic_apartments(
```

Comment:
What for this has done?

Reply:
The data has to be changed in tests to check that new functionality works when flags for publishing to Oikotie/Etuovi changes.